CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Precise UI Changelog
 
+## 0.5.1
```

Comment:
There is no need to make a changelog entry - we havent published 0.5 yet (pre-release).